sentry/README.md

```diff
@@ -50,35 +58,57 @@ one should use the [`sentry-core`] crate instead.
 
 ## Features
 
-Functionality of the crate can be turned on and off by feature flags.  This is the current list
-of feature flags:
-
-Default features:
-
-* `backtrace`: Enables backtrace support.
-* `contexts`: Enables capturing device, os, and rust contexts.
-* `panic`: Enables support for capturing panics.
-* `transport`: Enables the default transport, which is currently `reqwest` with `native-tls`.
-
-Additional features:
-
-* `anyhow`: Enables support for the `anyhow` crate.
-* `debug-images`: Attaches a list of loaded libraries to events (currently only supported on unix).
-* `error-chain`: Enables support for the `error-chain` crate.
-* `failure`: Enables support for the `failure` crate.
-* `log`: Enables support for the `log` crate.
-* `env_logger`: Enables support for the `log` crate with additional `env_logger` support.
-* `slog`: Enables support for the `slog` crate.
-* `test`: Enables testing support.
-* `debug-logs`: Uses the `log` crate for internal logging.
-* `reqwest`: Enables the `reqwest` transport, which is currently the default.
-* `curl`: Enables the curl transport.
-* `surf`: Enables the surf transport.
-* `native-tls`: Uses the `native-tls` crate, which is currently the default.
-  This only has an effect on the `reqwest` transport.
-* `rustls`: Enables the `rustls` support of the `reqwest` transport.
-  Please note that `native-tls` is a default feature, and one needs to use
-  `default-features = false` to completely disable building `native-tls` dependencies.
+Additional functionality and integrations are enabled via feature flags. Some features require
+extra setup to function properly. 
+
+| Feature | Default | Is Integration | Deprecated | Additional notes |
+| --- | --- | --- | --- | --- |
+| `backtrace` | ✅ | 🔌 | | |
+| `contexts` | ✅ | 🔌 | | |
+| `panic` | ✅ | 🔌 | | |
+| `transport` | ✅ | | | |
+| `anyhow` | | 🔌 | | |
+| `test` | | | | |
+| `debug-images` | | 🔌 | | |
+| `log` | | 🔌 |  |Requires additional setup; See [`sentry-log`]'s documentation. |
+| `debug-logs` | | | ✂️ | | Requires additional setup; See [`sentry-log`]'s documentation. |
```

Comment:
This is not really an integration per-se, it is rather a compile-time switch to have sentry output some internal logs to the `log` crate.